make/test.mk

```diff
@@ -160,7 +160,7 @@ ifneq ($(CLONEREFS_OPTIONS),)
 		echo "##########################################################################################################"; \
 		exit 1; \
 	fi; \
-	BRANCH_REF=`${GET_BRANCH_NAME} | awk '{print $$2}')`; \
```

Comment:
Sorry for this typo :-(

Reply:
Next time: https://github.com/sstephenson/bats